### PR TITLE
Remove HTML Encoding from output text of multiple Goodies (WIP)

### LINKED ISSUE #3563: Remove `html_enc` from Goodies. No longer needed :)
for fixing bug #3563

### PR DESCRIPTION
**Pull request by sahildua2305**
###### What does your Pull Request do (check all that apply)?

Choose the most relevant items and use the following title template to name
your Pull Request.

- [ ] New Instant Answer
    - [ ] Cheat Sheets: **`New {Cheat Sheet Name} Cheat Sheet`**
    - [ ] Other: **`New {IA Name} Instant Answer`**
- [x] Improvement
    - [x] Bug fix: **`{IA Name}: Fix {Issue number or one-line description}`**
    - [ ] Enhancement: **`{IA Name}: {Description of Improvements}`**
- [ ] Non–Instant Answer
    - [ ] Other (Role, Template, Test, Documentation, etc.): **`{GoodieRole/Templates/Tests/Docs}: {Short Description}`**

###### Description of changes

Provide an overview of the changes this pull request introduces.

###### Which issues (if any) does this fix?

Fixes #3563 

###### People to notify (@mention interested parties)
@moollaza

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: lib/DDG/Goodie/MD4.pm [hunk 1]
```diff
@@ -35,8 +35,8 @@ handle remainder => sub {
 
     return $out, structured_answer => {
         data => {
-            title => html_enc($out),
-            subtitle => html_enc('MD4 ' . $enc . ' hash').": " .html_enc($str)
+            title => $out,
+            subtitle => "MD4 " . $enc . " hash: $str"
```
Comment (GuiltyDolphin) [inline lib/DDG/Goodie/MD4.pm]: `"MD4 $enc hash: $str"`

File: lib/DDG/Goodie/NameDays.pm [hunk 2]
```diff
@@ -91,7 +91,7 @@ handle remainder => sub {
         # Search by name first
         $query = ucfirst($_);
         ($text, $html) = split('\|', $dates{lc($_)});
-        $header = 'Name days for <b>' . html_enc($query) . '</b>';
+        $header = 'Name days for <b>' . $query . '</b>';
```
Comment (GuiltyDolphin) [inline lib/DDG/Goodie/NameDays.pm]: `"Name days for <b>$query</b>"` (check others too).

**Comment by sahildua2305** [comment]
@GuiltyDolphin I have made the suggested changes. 

/cc @moollaza

**Comment by GuiltyDolphin** [comment]
@sahildua2305 Thanks!

**Pull request merged**